src/js/modules/streams/views/streams.item.event.tpl.html

```diff
@@ -31,7 +31,15 @@ <h2 class="page-title">
 			<th>Timestamp</th>
 		</tr>
 	</thead>
-	<tbody>
+	<tbody ng-hide="evt.eventNumber">
+		<tr>
+			<td colspan="5">
+				<em>Event does not exist for stream: {{streamId}}
```

Comment:
This is just a nitpick, but it might be nice to include the event number in the error message:
```suggestion
				<em>Event {{evt.eventNumber}} does not exist for stream: {{streamId}}
```

Reply:
Sure. I will add this and upload the changes